tutorials/btp-cf-buildpacks-java-create/btp-cf-buildpacks-java-create.md

```diff
@@ -0,0 +1,664 @@
+
+    ```JSON
+    "scripts": {
+        "start": "node node_modules/@sap/approuter/approuter.js"
+    },
+    ```
+
+10.	Now you need to add the `web` application to your project and bind the XSUAA service instance (`javauaa`) to it. To do that, insert the following content **at the end** of your `manifest.yml` file.
+
+
+    ```YAML
+    - name: web
+      random-route: true
+      path: web
+      memory: 1024M
+      env:
+        destinations: >
+          [
+            {
+              "name":"helloworld",
+              "url":"https://helloworld-1234-noway.cfapps.eu20.hana.ondemand.com/",
+              "forwardAuthToken": true
+            }
+          ]
+      services:
+      - javauaa
+    ```
+
+    > For the `url` parameter, use the generated URL (the `route` value for the `helloworld` application) from the previous step!
+
+11.	In the `web` folder, create an `xs-app.json` file with the following content:
+
+    ```JSON
+    {
+      "routes": [
+        {
+          "source": "^/helloworld/(.*)$",
+          "target": "$1",
+          "destination": "helloworld"
+        }
+      ]
+    }
+    ```
+
+    With this configuration, the incoming request is forwarded to the `helloworld` application, configured as a destination. By default, every route requires OAuth authentication, so the requests to this path will require an authenticated user.
+
+
+12. Open your `pom.xml` file and replace the entire `<dependencies>` block with the following:
+
+    ```XML
+    <dependencies>
+        <!-- Spring Boot starter packages -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-cloud-connectors</artifactId>
+            <version>2.2.13.RELEASE</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <!-- Spring and XSUAA Security -->
+        <dependency>
+            <groupId>com.sap.cloud.security.xsuaa</groupId>
+            <artifactId>xsuaa-spring-boot-starter</artifactId>
+            <version>2.12.2</version>
+        </dependency>
+
+        <!-- dependencies for test -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    ```
+
+
+13. Now go to the `java-tutorial` directory and execute:
+
+    ```Bash/Shell
+    mvn clean install
+    ```
+
+14. Then execute:
+
+    ```Bash/Shell
+    cf push
+    ```
+
+    This command will update the `helloworld` application and deploy the `web` application.
+
+    > ### What's going on?
```

Comment:
Why the ###?

Reply:
For consistency with the other two tutorials - [Python](https://developers.sap.com/tutorials/btp-cf-buildpacks-python-create.html) and [Node.js](https://developers.sap.com/tutorials/btp-cf-buildpacks-node-create.html)

For all of them, it's nice to emphasize on the change of behavior "from now on".